src/go/cmd/content.go

```diff
@@ -123,7 +123,8 @@ func includesType(s []string, t string) bool {
 	return false
 }
 
-func (c *content) parseSimpleType(tokenList *[]Token) {
+func (c *content) parseSimpleType() []ReviewLine {
```

Comment:
At some point we should add doc comments to the various funcs/methods. Might help future maintenance.